File: Tarea2_Dato_Abstracto_28-08-25/LEM_Dato_Abstracto.cs
using System;

class persona {
    public string Nombres;
    public string Ape1;
    public string Ape2;
}

class Program {
    static void Main() {
        persona p = new persona();
        p.Nombres = "Luis Eduardo";
        p.Ape1 = "Manzanarez";
        p.Ape2 = "Ramos";

        Console.WriteLine($"{p.Nombres} {p.Ape1} {p.Ape2}");
    }
}

File: Tarea2_Dato_Abstracto_28-08-25/LEM_Dato_Abstracto.cpp
#include <iostream>
#include <string>
using namespace std;

struct persona {
    string nombres;
    string ape1;
    string ape2;
};

int main() {
    persona p;
    p.nombres = "Luis Eduardo";
    p.ape1 = "Manzanarez";
    p.ape2 = "Ramos";

    cout << p.nombres << " " << p.ape1 << " " << p.ape2 << endl;
    return 0;
}